configs/abcxyz/github-token-minter.yaml

```diff
@@ -1,6 +1,6 @@
 # abcxyz/github-token-minter.yaml
 - 
   if: 'assertion.workflow_ref.startsWith("abcxyz/github-token-minter/.github/workflows/ci.yml")'
-  respositories: ['github-token-minter']
+  repositories: ['github-token-minter']
```

Comment:
stick with expanded form when not empty:

```
repositories: 
  - 'github-token-minter'
```

Reply:
ack